CONTRIBUTING.md

```diff
@@ -0,0 +1,68 @@
+# On Github Issues and Pull Requests
```

Comment:
Do we want to make an explicit note here about implementation help? A good percentage of Keras issues seem to be requests for help.